src/progress.rs

```diff
@@ -26,10 +26,11 @@
 // limitations under the License.
 
 use errors::Error;
-use fxhash::{FxBuildHasher, FxHashMap, FxHashSet};
+use hashbrown::{HashMap, HashSet};
+type DefaultHashMapBuilder = ::hashbrown::hash_map::DefaultHashBuilder;
+type DefaultHashSetBuilder = ::hashbrown::hash_map::DefaultHashBuilder;
```

Comment:
I'd prefer not to have type aliases in the import list, to either turn this into a `use .. as` statement or move it below the imports. I wouldn't hold up the PR on it though since it's already been in review a long time.